app/models/mixins/custom_actions_mixin.rb

```diff
@@ -48,6 +48,6 @@ def serialize_button(button)
   end
 
   def generic_custom_buttons
-    raise "called abstract method generic_custom_buttons"
+    CustomButton.buttons_for(self.class.name)
```

Comment:
@imtayadeway Wanted to get your thoughts on this change.  Previously only 3 files used this mixin and 2 need to specify other classes (Service -> ServiceTemplate, GenericObject -> GenericObjectDefinition).

Now we want to support this on all object that are exposed to custom buttons ([see list here](https://github.com/ManageIQ/manageiq/blob/master/app/models/custom_button.rb#L21-L50)) and now only those two exceptions above would need to override the `generic_custom_buttons` button.

Seems a much better approach then having to define `generic_custom_buttons` on all the classes that use it.

Reply:
This is good with me 👍